pkg/kube/utils.go

```diff
@@ -83,3 +95,44 @@ func GetPodContainerFromStatefulSet(ctx context.Context, cli kubernetes.Interfac
 	}
 	return podName, container[0].Name, nil
 }
+
+// Following functions get the regions and zones from
+// labels on Nodes and Persistent Volumes
+// As of kubernetes 1.17 the "failure.domain" annotation
+// has been deprecated in favor of the "topology" annotation
+
+func GetZoneFromNode(node v1.Node) string {
+	return GetZoneFromLabels(node.Labels)
+}
+
+func GetZoneFromPV(pv v1.PersistentVolume) string {
+	return GetZoneFromLabels(pv.Labels)
+}
+
+func GetZoneFromLabels(labels map[string]string) string {
```

Comment:
Just confirming - these are labels, not annotations, right?

Reply:
yes these are labels